aomi/cli.py

```diff
@@ -320,45 +368,56 @@ def template_runner(client, parser, args):
 
 def ux_actions(parser, args):
     """Handle some human triggers actions"""
-    if args.operation == 'help':
-        help_me(parser, args)
-
     # cryptorito uses native logging (as aomi should tbh)
-
     normal_fmt = '%(message)s'
-    if args.verbose == 2:
```

Comment:
Kind of? It now at least doens't throw errors if you aren't asking for verbosity...